test/test_helper.exs

```diff
@@ -14,7 +14,7 @@ alias Ecto.Integration.TestRepo
 Application.put_env(:ecto, TestRepo,
                     adapter: Mongo.Ecto,
                     url: "ecto://localhost:27017/ecto_test",
-                    pool_size: 1)
+                    size: 1)
```

Comment:
Okay, we can do the same in the driver.

On Wed, Jul 29, 2015 at 11:13 AM, Michał Muskała <notifications@github.com>
wrote:

> In test/test_helper.exs
> <https://github.com/michalmuskala/mongodb_ecto/pull/35#discussion_r35740758>
> :
>
> > @@ -14,7 +14,7 @@ alias Ecto.Integration.TestRepo
> >  Application.put_env(:ecto, TestRepo,
> >                      adapter: Mongo.Ecto,
> >                      url: "ecto://localhost:27017/ecto_test",
> > -                    pool_size: 1)
> > +                    size: 1)
>
> I'm using the driver's adapters. Ecto simply translates pool_size to size
> before handing options to poolboy.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/michalmuskala/mongodb_ecto/pull/35/files#r35740758>.
>



-- 
Eric Meadows-Jönsson